README.md

```diff
@@ -338,49 +338,44 @@ This wil load `htmldev/components/cards/cover.html.twig` with the given properti
 
 #### Rendering SVG files
 
-The HtmldevBundle provides an `svg` macro to inline render SVG's located in `htmldev`. This allows
-easy coloring of the SVG with `currentColor` and CSS.
+*The HtmldevBundle `svg` macro (`ui.svg()`) to inline render SVG's is deprecated*
+
+The HtmldevBundle provides a `inline_images` filter to inline render SVG's within the string subject.
+This allows easy coloring of the SVG with `currentColor` and CSS.
 
 This will render the contents of `htmldev/images/icons/arrow--right.svg` in the HTML:
 
 ```twig
-{% import 'ZichtHtmldevBundle:macros:components.html.twig' as ui %}
-
-{{ ui.svg('arrow--right', { 
-    width: 20,
-    height: 20,
-    directory: 'images/icons'
-}) }}
-``` 
+{% apply inline_images %}
```

Comment:
Waarom heet het `inline_images`? Kan er buiten het wergeven van een SVG nog meer mee? :)

Reply:
Er zit inderdaad een `switch($extensie)` in met een `case '.svg'`. Hier zou ook een PNG verwerker bij kunnen die er dan bijv. een `<img src="data:image/png;base64,......." />` van maakt (en Gif en JPEG bijv. ook)

Reply:
Check, nice :) Als het alleen voor svg's had geweest had de naam die kant op kunnen wijzen, maar prima zo :)